src/components/Button.jsx

```diff
@@ -1,18 +1,16 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable arrow-body-style */
 /* eslint-disable react/prop-types */

```

Comment:
- [ ] Please don't disable any linter rule. You can fix all of these properly. Remember that linters are checks to make sure that code within organizations remains consistent and clean. It's important that you learn how to solve these issues as you will likely encounter many more once you are hired. [You can find information about how to construct PropTypes for React here.](https://reactjs.org/docs/typechecking-with-proptypes.html)